### ISSUE TITLE
tinaface，eval result doesn't have esay-medium-hard result?

### ISSUE THREAD

**Issue by wang-kangkang** [Media-Smart/vedadet#62]
跑tinaface的评测，只有voc的ap，没有easy，medium，hard的结果：
+-------+-------+----------+--------+-------+
| class | gts   | dets     | recall | ap    |
+-------+-------+----------+--------+-------+
| face  | 31957 | 65363592 | 0.995  | 0.916 |
+-------+-------+----------+--------+-------+
| mAP   |       |          |        | 0.916 |
+-------+-------+----------+--------+-------+
我想要easy medium hard的结果该怎么弄？

**Comment by hxcai** [comment]
@wang-kangkang please see the [link](https://github.com/Media-Smart/vedadet/tree/main/configs/trainval/tinaface#evaluation) to get easy, medium, hard reuslts.

**Comment by wang-kangkang** [comment]
> @wang-kangkang please see the [link](https://github.com/Media-Smart/vedadet/tree/main/configs/trainval/tinaface#evaluation) to get easy, medium, hard reuslts.

我就是照着这个做的，才得到上面没easy-medium-hard的这种结果，能再说详细些吗？你是说跑那个matlab代码吗？
i was did by your link, and than get my result.  can you be more specific? you mean the matlab code in eval_tools?

**Comment by hxcai** [comment]
@wang-kangkang Yes, you need to run the official matlab code in eval_tools to get results.

**Comment by wang-kangkang** [comment]
> @wang-kangkang Yes, you need to run the official matlab code in eval_tools to get results.

thank you.
i have another question. the result in wider_val hard is 0.934 or 0.924 in picture, but the result in wider_val hard is 0.930 or 0.922 in table. why are they different?

**Comment by hxcai** [comment]
@wang-kangkang The results in table are without TTA and the results in picture are our final score with TTA.

**Comment by wang-kangkang** [comment]
> @wang-kangkang The results in table are without TTA and the results in picture are our final score with TTA.

thank you
I have the third question. in test_widerface.py, i find the third param in build_dataloader() need to be 0, if use 1 by default, it will report segment fault. do you know why? the time of predict all Wider_val need almost 2 hours

**Comment by hxcai** [comment]
@wang-kangkang We haven't meet the error.
